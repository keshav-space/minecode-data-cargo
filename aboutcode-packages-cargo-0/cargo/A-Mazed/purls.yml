- pkg:cargo/A-Mazed@0.1.0
