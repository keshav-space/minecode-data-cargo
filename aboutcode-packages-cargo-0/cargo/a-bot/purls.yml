- pkg:cargo/a-bot@0.0.1
- pkg:cargo/a-bot@0.1.0
- pkg:cargo/a-bot@0.1.1
