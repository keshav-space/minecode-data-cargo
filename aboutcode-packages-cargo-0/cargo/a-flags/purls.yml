- pkg:cargo/a-flags@0.1.0
- pkg:cargo/a-flags@0.1.1
- pkg:cargo/a-flags@0.1.2
- pkg:cargo/a-flags@0.1.3
- pkg:cargo/a-flags@0.2.0
