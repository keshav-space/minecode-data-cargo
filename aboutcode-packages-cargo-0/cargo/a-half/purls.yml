- pkg:cargo/a-half@0.1.0
