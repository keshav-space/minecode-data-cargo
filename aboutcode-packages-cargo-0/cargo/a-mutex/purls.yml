- pkg:cargo/a-mutex@0.1.0
- pkg:cargo/a-mutex@0.1.1
- pkg:cargo/a-mutex@0.1.2
