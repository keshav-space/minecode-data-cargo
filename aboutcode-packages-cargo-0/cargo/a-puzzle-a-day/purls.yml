- pkg:cargo/a-puzzle-a-day@0.1.0
- pkg:cargo/a-puzzle-a-day@0.1.1
