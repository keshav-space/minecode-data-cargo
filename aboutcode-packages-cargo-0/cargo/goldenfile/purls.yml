- pkg:cargo/goldenfile@0.4.0
- pkg:cargo/goldenfile@0.5.0
- pkg:cargo/goldenfile@0.5.1
- pkg:cargo/goldenfile@0.6.0
- pkg:cargo/goldenfile@0.7.0
- pkg:cargo/goldenfile@0.7.1
- pkg:cargo/goldenfile@1.0.0
- pkg:cargo/goldenfile@1.0.1
- pkg:cargo/goldenfile@1.0.2
- pkg:cargo/goldenfile@1.0.3
- pkg:cargo/goldenfile@1.1.0
- pkg:cargo/goldenfile@1.2.0
- pkg:cargo/goldenfile@1.3.0
- pkg:cargo/goldenfile@1.4.0
- pkg:cargo/goldenfile@1.4.1
- pkg:cargo/goldenfile@1.4.2
- pkg:cargo/goldenfile@1.4.3
- pkg:cargo/goldenfile@1.4.4
- pkg:cargo/goldenfile@1.4.5
- pkg:cargo/goldenfile@1.5.0
- pkg:cargo/goldenfile@1.5.1
- pkg:cargo/goldenfile@1.5.2
- pkg:cargo/goldenfile@1.6.0
- pkg:cargo/goldenfile@1.6.1
- pkg:cargo/goldenfile@1.7.0
- pkg:cargo/goldenfile@1.7.1
- pkg:cargo/goldenfile@1.7.2
- pkg:cargo/goldenfile@1.7.3
- pkg:cargo/goldenfile@1.8.0
